src/aliases/api/handlers/utils.go

```diff
@@ -0,0 +1,68 @@
+package handlers
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/consensys/quorum-key-manager/pkg/errors"
+)
+
+func jsonWrite(w http.ResponseWriter, data interface{}) error {
+	w.Header().Set("Content-Type", "application/json; charset=UTF-8;")
+	w.Header().Set("X-Content-Type-Options", "nosniff")
+	return json.NewEncoder(w).Encode(data)
+}
+
+func WriteHTTPErrorResponse(rw http.ResponseWriter, err error) {
+	var writeErr error
+	switch {
+	case errors.IsAlreadyExistsError(err):
+		writeErr = writeErrorResponse(rw, http.StatusConflict, err)
+	case errors.IsNotFoundError(err):
+		writeErr = writeErrorResponse(rw, http.StatusNotFound, err)
+	case errors.IsUnauthorizedError(err):
+		writeErr = writeErrorResponse(rw, http.StatusUnauthorized, err)
+	case errors.IsInvalidFormatError(err):
+		writeErr = writeErrorResponse(rw, http.StatusBadRequest, err)
+	case errors.IsInvalidParameterError(err), errors.IsEncodingError(err):
+		writeErr = writeErrorResponse(rw, http.StatusUnprocessableEntity, err)
+	case errors.IsNotImplementedError(err), errors.IsNotSupportedError(err):
+		writeErr = writeErrorResponse(rw, http.StatusNotImplemented, err)
+	default:
+		writeErr = writeErrorResponse(rw, http.StatusInternalServerError, fmt.Errorf(internalErrMsg))
+	}
+	if writeErr != nil {
+		// TODO the: use logger
+		log.Printf("error writing the original error: %v: %v", writeErr, err)
+		http.Error(rw, writeErr.Error(), http.StatusInternalServerError)
+	}
+}
+
+func writeErrorResponse(w http.ResponseWriter, status int, err error) error {
```

Comment:
Same about moving common http code to `/src/infra/http` and reuse it from there

Reply:
done already in another PR.